src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -141,11 +146,25 @@ def password_correct?
         end
 
         # Password is considered strong when cracklib returns an empty message.
+        # User has the last word to decide whether to use a weak password.
         def password_strong?
           message = cracklib_message
           return true if message.empty?
-          Yast::Report.Warning(message)
-          false
+
+          popup_text = [
+            _("The password is too simple:"),
```

Comment:
I would prefer what you propose: "The password is too weak". But I have used "too simple" to mimic this: https://openqa.opensuse.org/tests/385493#step/user_settings/5

So, the question is whether continue using the same than before or not.